bower.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "ember-cli-github-pages",
   "dependencies": {
-    "handlebars": "~1.3.0",
-    "jquery": "^1.11.1",
-    "ember": "1.8.1",
-    "ember-data": "1.0.0-beta.12",
-    "ember-resolver": "~0.1.10",
-    "loader.js": "stefanpenner/loader.js#1.0.1",
-    "ember-cli-shims": "stefanpenner/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "rwjblue/ember-cli-test-loader#0.0.4",
-    "ember-load-initializers": "stefanpenner/ember-load-initializers#0.0.2",
-    "ember-qunit": "0.1.8",
-    "ember-qunit-notifications": "0.0.4",
-    "qunit": "~1.15.0"
+    "ember": "2.0.1",
+    "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
+    "ember-cli-test-loader": "0.2.0",
+    "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
+    "ember-qunit": "0.4.10",
+    "ember-qunit-notifications": "0.0.7",
+    "jquery": "^1.11.3",
+    "loader.js": "3.3.0",
+    "qunit": "~1.18.0"
+  },
+  "resolutions": {
+    "ember": "2.0.1"
```

Comment:
Not sure if we want to insist on addon development being done w/ ember 2.0. It might be a little annoying for people to catch surprise 1.x compatibility issues after opening a PR.

I like to keep addon "demo app" dependencies at the oldest ember version the addon aims to support. Usually this means that when things pass locally, they'll also pass in CI without incident.